lib/bot.js

```diff
@@ -40,6 +42,9 @@ function Bot(options) {
 Bot.prototype.attachListeners = function() {
   this.client.addListener('registered', function(message) {
     logger.debug('Registered event: ', message);
+    for (var i = 0, len = this.autosendcmd.length; i < len; i++) {
```

Comment:
Yep, but you can pass `this` as the second argument to `forEach` instead:
```js
    this.autoSendCommands.forEach(function(command) {
      this.client.send.apply(this.client, command);
    }, this);
```
Also you don't need to specify all three `forEach` arguments, since you're only using the first :)